crm_facebook_leads/i18n/es.po

```diff
@@ -4,22 +4,34 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 14.0+e\n"
+"Project-Id-Version: Odoo Server 15.0+e\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-12-10 02:32+0000\n"
-"PO-Revision-Date: 2020-12-10 02:32+0000\n"
-"Last-Translator: \n"
+"POT-Creation-Date: 2022-06-07 01:58+0000\n"
+"PO-Revision-Date: 2022-06-07 01:58+0000\n"
+"Last-Translator: Luis González <lgonzalez@vauxoo.com>\n"
 "Language-Team: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: \n"
 "Plural-Forms: \n"
 
+#. module: crm_facebook_leads
+#: model:ir.model.fields,field_description:crm_facebook_leads.field_utm_adset__id_facebook_adset
+msgid "Adset ID"
+msgstr "ID de grupo de anuncios"
+
 #. module: crm_facebook_leads
 #: model:ir.model.fields,field_description:crm_facebook_leads.field_crm_facebook_form__allow_to_sync
 msgid "Allow To Sync"
 msgstr "Permitir Sincronización"
 
+#. module: crm_facebook_leads
+#: model:ir.actions.server,name:crm_facebook_leads.crm_facebook_cron_ir_actions_server
+#: model:ir.cron,cron_name:crm_facebook_leads.crm_facebook_cron
+#: model:ir.cron,name:crm_facebook_leads.crm_facebook_cron
+msgid "CRM Facebook Lead Ads: Fetch leads"
+msgstr "CRM Facebook Lead Ads: Obtener anuncios "
```

Comment:
`anuncios` it's not a good translation to `leads`

```suggestion
msgstr "CRM Facebook Lead Ads: Obtener Clientes Potenciales"
```
or 
```suggestion
msgstr "CRM Facebook Lead Ads: Obtener Iniciativas"
```

Reply:
Changed by iniciativas, which is the native translation for leads